pkg/flowaggregator/flowaggregator.go

```diff
@@ -217,28 +298,27 @@ func (fag *flowAggregator) Run(stopCh <-chan struct{}) {
 	}
 }
 
-func (fag *flowAggregator) sendFlowKeyRecords(key ipfixintermediate.FlowKey, records []ipfixentities.Record) error {
+func (fag *flowAggregator) sendFlowKeyRecords(key ipfixintermediate.FlowKey, record ipfixintermediate.AggregationFlowRecord) error {
 	setSent := ipfix.NewSet(ipfixentities.Data, fag.templateID, false)
```

Comment:
This can be moved outside the callback right? By doing that, we can support multiple records in one data set. Is there a plan to do this in a follow-up patch?

Reply:
Sure. updated.